cli/Cargo.toml

```diff
@@ -20,7 +20,7 @@ exonum-supervisor = { version = "1.0.0-rc.1", path = "../services/supervisor" }
 exonum-system-api = { version = "1.0.0-rc.1", path = "../components/system-api" }
 exonum-rust-runtime = { version = "1.0.0-rc.1", path = "../runtimes/rust" }
 
-failure = "0.1"
+failure = { package = "anyhow", version = "1.0" }
```

Comment:
Nice hack 🥇 

But are you sure that it doesn't lead to misunderstandings in the end users code?